### DISCUSSION TITLE
Setting page (edit/view/etc) permissions (newbie question)

### DISCUSSION THREAD

**Discussion by DrOteonu-Nanquanu** [requarks/wiki#7067] [Help / Questions]
We are currently migrating from Tiki to Wiki.js.

- On Tiki you set page permissions on who can view/edit the page, on the page itself. I.e. on each page, there is a permission button - when clicked you can fine-regulate permission to that specific page.
- How is this done in wiki.js? My impression is that it works differently. Indirectly: you can only set permissions on directories, not on pages. Then all pages placed in that directory inherit the permissions of the dir. Is that right?
- Or can you also do it in the way Tiki does it?

**Comment by fnafox** [comment]
You can add permissions on a per page basis by giving groups permissions to exact paths.
Admin > Groups > Edit group... then under page rules you can add permissions on whatever context and with the level of granularity you wish.

**Reply by dogweather** [reply]
I'm looking for the list of permissions that can be given to groups. Is it documented somewhere?

**Reply by fnafox** [reply]
There's documentation here: https://docs.requarks.io/groups
full list of permissions is available in the admin interface

**Reply by dogweather** [reply]
@fnafox Thanks, I read that. I'm figuring out if wiki.js will meet my needs. 

So I'm looking in the source for the list of permissions _before_ I install.
